packages/botonic-react/src/components/custom-message.tsx

```diff
@@ -6,7 +6,7 @@ import { SENDERS } from '../index-types'
 import { createErrorBoundary } from '../util/error-boundary'
 import { warnDeprecatedProps } from '../util/logs'
 import { mapObjectNonBooleanValues } from '../util/react'
-import { CustomMessageType } from './index-types'
+import { CustomMessageType } from '../webchat/theme/types'
```

Comment:
I think that this type should not go within webchat/theme/types but in another file. But ok for this moment

Reply:
It is only used in the Theme that's why I brought it to this file but I agree that it doesn't make much sense. I'll move it to src/components/index-types and we'll see where it needs to be later.